packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-node-julia.vue

```diff
@@ -14,19 +24,44 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from 'vue';
+import { ref, computed, watch } from 'vue';
 import Button from 'primevue/button';
 import TeraOperatorPlaceholder from '@/components/operator/tera-operator-placeholder.vue';
+import TeraSimulateChart from '@/workflow/tera-simulate-chart.vue';
 import { WorkflowNode } from '@/types/workflow';
+import { RunResults } from '@/types/SimulateConfig';
+import { getRunResult } from '@/services/models/simulation-service';
+import { csvParse } from 'd3';
 import { SimulateJuliaOperationState } from './simulate-julia-operation';
 
 const props = defineProps<{
 	node: WorkflowNode<SimulateJuliaOperationState>;
 }>();
 
 const emit = defineEmits(['open-drilldown']);
+const runResults = ref<RunResults>({});
+const hasData = ref(false);
 
 const areInputsFilled = computed(() => props.node.inputs[0].value);
+
+const selectedRunId = ref<any>(null);
+
+watch(
+	() => props.node.active,
+	async () => {
+		const active = props.node.active;
+		if (!active) return;
+
+		selectedRunId.value = props.node.outputs.find((o) => o.id === active)?.value?.[0];
+		if (!selectedRunId.value) return;
```

Comment:
Should we set hasData.value back to false here?
Waiting to run this locally on pulling but it looks like we may run into a scenario with output switching where we are showing old results? 

I may be mistaken I have not yet run this

Reply:
Probably, but can be cleaned up later.

Reply:
Very fair